### ISSUE TITLE
Custom artist cover art

### ISSUE THREAD

**Issue by magnuslarsen** [jaedb/Iris#796]
### Problem to solve

Note: I _only_ use local music.

On the "Artists" page, cover art is [loaded from Discogs](https://github.com/jaedb/Iris/blob/master/src/js/services/discogs/actions.js) - which for some edge cases retrieves the wrong artist (e.g. [here where the artist](https://www.discogs.com/search/?q=K%2FDA&type=all) `K. Da Cruz` comes before the intended search `K/DA`)

It would be cool if we could specify custom artist art, where such case happens.

### Proposed solution

I know [that Jellyfin calls](https://jellyfin.org/docs/general/server/media/music.html#images) their "artist cover art" files `cover.jpg`, and is placed in the root of the artist folder:
```
mediatv@mediatv ~/M/M/K_DA> ls -lh
total 632K
drwxrwxr-x  2 mediatv mediatv  208 Oct 20 18:07 'ALL OUT'/
-rw-r--r--  1 mediatv mediatv  555 Jan 22 13:17  artist.nfo
-rw-r--r--  1 mediatv mediatv 190K Oct 20 18:07  fanart.jpg
-rw-r--r--  1 mediatv mediatv 139K Oct 20 18:07  folder.jpg
-rw-r--r--  1 mediatv mediatv 292K Oct 20 18:07  logo.png
drwxrwxr-x  3 mediatv mediatv   72 Jan  9 13:16  POP_STARS/
```

Make a default check for that file? I'm not too sure if other popular organizers handles this, but I'd reckon it's fairly standard.

Alternatively right-clicking the artist should give a "change artist cover" option, where you could specify a URL (`http(s)://` or `file://`), and store that in some database / Iris working directory

**Comment by sjbat** [comment]
+1

**Comment by fmarzocca** [comment]
+1

**Comment by jaedb** [comment]
Mopidy does not currently support artist artwork, which would be the best way for Iris to read files from your local library. Because of this, the best way forward is to explore how the Discogs lookup can better handle non-alphanumeric characters.

**Comment by magnuslarsen** [comment]
Thanks for the feedback :-) Unfortunately it isn't _just_ alphanumeric characters that are the issue; some artists just have too generic names, that it becomes impossible to create a "good" query.

E.g. [see this search for "Jennie"](https://www.discogs.com/search/?q=jennie&type=all) - the artist intended `Jennie` is at 4th place, behind a lot of Jennie's

**Comment by kingosticks** [comment]
Does discogs support musicbrainz IDs?

**Comment by kingosticks** [comment]
Also, feel free to open an issue at Mopidy (or Mopidy-Local?) if there isn't one already for this.

**Comment by jaedb** [comment]
@kingosticks I don't believe it does allow querying by Musicbrainz ID unfortunately. I would be using Musicbrainz instead if only they provided Artist artwork...

If Discogs is pulling in incorrect images relatively consistently then my preference would be to ditch artist artwork all together. Instead Iris would show the artwork for the first album. Boring is better than wrong.
